test/pieces.js

```diff
@@ -674,30 +689,30 @@ describe('Pieces', function() {
     assert(response.counts.articles[0].value === 'first-article');
   });
 
-  it('can patch a join', async () => {
+  it('can patch a relationship', async () => {
     let response = await apos.http.post('/api/v1/article', {
       jar,
       body: {
-        title: 'Join Article',
-        name: 'join-article'
+        title: 'Relationship Article',
```

Comment:
"Related Article," he suggested pedantically. Optional.

Reply:
Leaving it because in this test it is literally for testing a feature by that name only. Otherwise would agree.